_data/toc.yaml

```diff
@@ -1327,14 +1327,14 @@ manuals:
           title: Quickstart
         - sectiontitle: "Part one: Build"
           section:
-            - sectiontitle: "Step one: Set up..."
+            - sectiontitle: "Step one: Create a new extension"
               section:
-                - title: ...a minimal frontend extension
+                - title: Create an simple extension
                   path: /desktop/extensions-sdk/build/set-up/minimal-frontend-extension/
-                - title: ... an advanced frontend extension
+                - title: Invoke Docker commands
```

Comment:
Something is wrong with this title, it's not about invoking Docker commands :) 

```suggestion
                - title: Create a more advanced front end
```

Reply:
Hum, yes you are right...